developer_manual/digging_deeper/reference.rst

```diff
@@ -226,6 +226,15 @@ For example, resolving ``https://www.themoviedb.org/movie/70981`` if the ``integ
       }
     }
 
+Render link previews in clients
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Clients can choose to support some rich objects types.
+Developers can follow the rich object formatting recommendations to provide generic information in some cases.
+The rich object type is not used to predict the data structure.
+We rather recommend to set rich object attributes respecting a strict format for some common use cases.
+
+More details can be found in `Provide generic data for clients <data-for-clients_>`__
```

Comment:
```suggestion
More details can be found in :ref:`data-for-clients`
```

This way when the section or label is adjusted the link will still work or throw an error on compile instead of silently linking to nothing.